src/firejail/util.c

```diff
@@ -45,6 +45,28 @@
 #define MAXBUF 4098
 #define EMPTY_STRING ("")
 
+int is_valid_unit(char c){
+	/* check char valid unit ( K or M or G ) */
+	if( c == 'K' || c == 'k' ) return 1;
+	if( c == 'M' || c == 'm' ) return 1;
+	if( c == 'G' || c == 'G' ) return 1;
```

Comment:
two times uppercase 'G'?

Reply:
you marked as resolved? there are still two 'G'. :-)

Reply:
oh, i pushed wrong commit. i fixed this now. sorry for this.